geonode/apps/thuenen_app/templatetags/gdi_de_custom_roles.py

```diff
@@ -0,0 +1,25 @@
+from django import template
+
+register = template.Library()
+
+GDI_DE_ROLE_LABEL_MAPPING = {
+    'Owner': 'owner',
+    'Point of Contact': 'pointOfContact',
+    'Metadata Author': 'author',
+    'Processor': 'processor',
+    'Publisher': 'publisher',
+    'Custodian': 'custodian',
+    'Distributor': 'distributor',
+    'Resource User': 'user',
+    'Resource Provider': 'resourceProvider',
+    'Originator': 'originator',
+    'Principal Investigator': 'principalInvestigator'
+}
+
+@register.filter
+def get_gdi_compliant_role_label(value):
```

Comment:
Could fix this upstream issue here: https://github.com/GeoNode/geonode/issues/12534

Reply:
If upstream wants this, please go ahead.